pkg/cmd/pr/create/create.go

```diff
@@ -412,19 +464,32 @@ func initDefaultTitleBody(ctx CreateContext, state *shared.IssueMetadataState, u
 	if err != nil {
 		return err
 	}
-	if len(commits) == 1 || useFirstCommit {
-		commitIndex := len(commits) - 1
-		state.Title = commits[commitIndex].Title
-		body, err := gitClient.CommitBody(context.Background(), commits[commitIndex].Sha)
-		if err != nil {
-			return err
-		}
-		state.Body = body
+
+	if useFirstCommit {
+		state.Title = commits[len(commits)-1].Title
+		state.Body = commits[len(commits)-1].Body
 	} else {
 		state.Title = humanize(headRef)
 		var body strings.Builder
 		for i := len(commits) - 1; i >= 0; i-- {
-			fmt.Fprintf(&body, "- %s\n", commits[i].Title)
+			fmt.Fprintf(&body, "- **%s**\n", commits[i].Title)
+			if addBody {
+				chunks := chunkString(commits[i].Body, 72)
+				for idx, chunk := range chunks {
+					if chunk[0] == ' ' {
+						chunk = chunk[1:]
+					}
+					fmt.Fprintf(&body, "  %s", chunk)
+					if idx < len(chunks)-1 {
+						fmt.Fprintln(&body)
+					}
+				}
```

Comment:
**What would happen if body was not chunked on 72 characters?**

Being the logic that adds body information to the PR body, this is indenting the commit body to ensure the output follows markdown formatting.  This chunking is effectively word wrapping the content before it is indented, correct?

Whether to word wrap this or not is a funny situation because this ideally should use the body as-is.  Instead of chunking, this could be simplified with something like:

```go
    var regexPattern = regexp.MustCompile(`(?m)^`)
    ...
    fmt.Fprintln(regexPattern.ReplaceAllString(commits[i].Body, "  "))
```

Reply:
Ok this make sense to me also! I fixed using your suggestion